dist/joii.js

```diff
@@ -1176,6 +1207,96 @@
             }
         }
 
+
+        /**
+         * Serializes all serializable properties of an object. Public members are serializable by default.
+         *
+         * @return JSON string containing the serialized class
+         */
+        definition.prototype.Serialize = function ()
```

Comment:
Applying 'standard' methods to the outer scope is effectively a BC-break. You never know if people have classes that already have their own methods with the same names, which you'd effectively overwrite with this commit.

As far as I see it, there are 3 options here:

1. Introduce a BC break, releasing JOII 4.x - not recommended.
2. Use the magic-prefix of a double underscore. Everybody hates it, nobody wants it, but it's there for the ones who want it. So the method would become `__serialize` and `__deserialize`.
3. Add the serialize/deserialize methods to the `JOII` scope, leaving class definitions alone.

The effect for 3 would be:

```js
var my_class = new MyClass('foo', 'bar');
var serialized_data = JOII.serialize(my_class);

var my_class = JOII.deserialize(MyClass, serialized_data);
```
My preference would go to the last one, since it shouldn't break any existing code.